src/external/wb_presets.c

```diff
@@ -3068,6 +3077,15 @@ const wb_data wb_preset[] =
   { "Nikon", "D850", DaylightFluorescent             , 0, { 2.018066406, 1, 1.112304688, 0 } },
   { "Nikon", "D850", HighTempMercuryVaporFluorescent , 0, { 2.379882812, 1, 1.3671875,   0 } },
 
+  /*NIKON D850 Software Ver.1.03*/
```

Comment:
You need to:
1. remove the old D850 entries
2. do not add the D500 in the same commit
3. rename the entries from ""NIKON CORPORATION" to "Nikon" and "NIKON D850" to "D850".

I have done 3 for the D500.

Thanks.

Reply:
1. Ok, I wasn't sure about that since somehow the number of entries differ. Will do that later.
2. D500 was in #2012 
3. I just used the output of extract_wb and already wondered about the difference. Will do that later today.